src/core-concepts-in-embedded-application/alloc.md

```diff
@@ -0,0 +1,59 @@
+# Alloc
```

Comment:
- This chapter is a how-to without really explaining anything, while the `async` chapter is explicitly claiming not to be a tutorial. This looks very odd to me.
- This chapter refers to modifying files, but what is the starting state? esp-generate gives the complete setup if asked for, what do people need to modify?
- PSRAM should be included here, I think.

Reply:
Agree with PSRAM. Also agree with 1st and 2nd points - but I'm not sure what to do here tbh. 
Do we want to explain something instead of providing "how-to"? What do we want to explain?

Reply:
I don't really know, it just feels out of place to me, as opposed to "book style". But I am not competent in this area :)